spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,137 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
+
+  describe 'GET users#index' do
+
+    before do
+      create(:user)
+      create(:user)
+    end
+
+    it 'returns succssfully' do
+      get :index
+      expect(response).to be_successful
+    end
+
+    it 'follows the routes successfully' do
+      get :index
```

Comment:
You can add this line in the before block.

Reply:
Done!